pkg/l2announcer/l2announcer.go

```diff
@@ -139,17 +139,18 @@ func NewL2Announcer(params l2AnnouncerParams) *L2Announcer {
 
 func (l2a *L2Announcer) run(ctx context.Context, health cell.Health) error {
 	var err error
-	l2a.svcStore, err = l2a.params.Services.Store(ctx)
+	l2a.policyStore, err = l2a.params.L2AnnouncementPolicy.Store(ctx)
```

Comment:
nit: Is there a reason why order was swapped? I guess not, but just double checking. 
Before, we had services -> policy, now we have policy -> service